packages/common/package.json

```diff
@@ -45,9 +45,8 @@
     "chalk-pipe": "^3.0.0",
     "dotenv": "^6.2.0",
     "eth-crypto": "^1.7.0",
-    "hardhat": "^2.0.4",
```

Comment:
I think we need to keep hardhat as a production dependency since it's required by files that are exported by the package. Otherwise, anyone who installs the package wouldn't automatically install hardhat and any `require("hardhat")` calls would fail on import.

Reply:
hmmm, do you think that's why some tests are failing? I'll try that

Reply:
Looks like a more recent `hardhat` change forces us to specify `transactionConfig.chainId` as a hex number instead of an int. Trying to find the PR that changed this